app/controllers/messages_controller.rb

```diff
@@ -27,6 +27,12 @@ def update
     redirect_back fallback_location: messages_path(@message)
   end
 
+  def export
+    authorize @message
+
+    send_data @message.prepare_message_export, filename: "#{@message.id}.zip", type: 'application/x-zip-compressed', disposition: :download
```

Comment:
Este uvazujem, ze to asi radsej nazvat podla `@message.title`.

Reply:
title neviem. to myslim je vacsinou bordel. skor by som zvazil nieco ako `yyyy-mm-dd-sprava-id`